ERCS/erc-7694.md

```diff
@@ -0,0 +1,144 @@
+---
+eip: 7694
+title: Solana Storage Router
+description: Cross-chain storage router protocol (CCIP-Store) incorporating storage router for Solana
+author: Avneet Singh (@sshmatrix), 0xc0de4c0ffee (@0xc0de4c0ffee)
+discussions-to: https://ethereum-magicians.org/t/erc-7694-solana-storage-router/19706
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-04-18
+---
+
+## Abstract
+The following standard is an extension to the cross-chain storage router protocol introducing storage router for Solana.
+
+## Motivation
+Cross-Chain Storage Router Protocol, aka CCIP-Store, introduces three external routers for routing storage to L1 contracts, L2s and databases. This document extends that specification by introducing a fourth storage router targeting Solana as the storage provider. 
```

Comment:
Is CCIP-Store defined in an EIP? If so, you should reference it. If not, please link to it according to the rules in [EIP-1](https://eips.ethereum.org/EIPS/eip-1#linking-to-external-resources).

Reply:
CCIP-Store is defined in open PR #404 which introduces EIP-7700. I have added the reference to it, although the CI checks will fail until PR #404 is merged and EIP-7700 becomes available.